ethminer/MinerAux.h

```diff
@@ -774,8 +777,12 @@ class MinerCLI
 				{
 					auto mp = f.miningProgress();
 					f.resetMiningProgress();
-					if (current)
+					if (current){
```

Comment:
Existing style throughout this project would appear to be opening braces on new line.